README.md

```diff
@@ -8,17 +8,18 @@ All it requires is that you have your [Vultr API key](https://my.vultr.com/setti
 
 These instructions assume you are on a shell as the `root` user.
 
-1. Download `vultr-dns.py` somewhere onto your server. In this example, we will use `/etc/letsencrypt/vultr-dns.py` as the location.
-2. `chmod 0700 /etc/letsencrypt/vultr-dns.py && chown root:root /etc/letsencrypt/vultr-dns.py`
-3. Modify the configuration section of `/etc/letsencrypt/vultr-dns.py` :
+1. Make sure you have the requirements `python2` and `python2-requests` installed. Obviously, you should also have `certbot` installed. (Note: On Debian systems the '2' is omitted from python2 package names)
```

Comment:
I think probably we should not specify the Python version here. Since the hook itself uses the system default Python version,  `python` and `python-requests` would point to the correct version anyway (for EL-based and Debian-based systems).

Maybe:

```suggestion
1. Make sure you have `python` and `python-requests` installed, in addition to Certbot.
```